test/empty-brace-spaces.mjs

```diff
@@ -22,7 +22,6 @@ const cases = [
 	'while (foo){/* */}',
 	'foo = () => {/* */}',
 	'foo = function (){/* */}',
-	'function foo(){/* */}',
```

Comment:
Please add this back.

Reply:
This is based on the message of `npm run run-rules-on-codebase`
This line is repeated on line 9
Why do I need to add it back？

Reply:
Sorry, didn't know that. Seems this rule already helping! 👍